package.json

```diff
@@ -17,6 +17,10 @@
     "test": "mocha -r ts-node/register src/**/*.spec.ts",
     "build": "tsc -p tsconfig.json",
     "build-dev": "tsc -p tsconfig.development.json",
+    "coverage": "nyc mocha -r ts-node/register src/**/*.spec.ts",
+    "generate-coverage-report": "nyc report --reporter=text-lcov > lcov.info",
+    "test-ci": "MOCHA_FILE=./reports/test-results.xml mocha -r ts-node/register src/**/*.spec.ts --reporter mocha-junit-reporter",
+    "test-console": "mocha -r ts-node/register src/**/*.spec.ts --reporter mocha-junit-reporter --reporter-options toConsole",
```

Comment:
I would remove this, as I think it was just for local testing.

Reply:
in: https://github.com/sonatype-nexus-community/auditjs/pull/173/commits/8400685999c790751b590ae174f33cb14f3f7788